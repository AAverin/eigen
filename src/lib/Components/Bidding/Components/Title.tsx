import React from "react"

import { SerifSemibold18 } from "../Elements/Typography"

export const Title = props => <SerifSemibold18 textAlign="center" {...props} />
