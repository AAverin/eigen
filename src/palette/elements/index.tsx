export * from "./Cards"
export * from "./Touchable"
export * from "./Typography"
