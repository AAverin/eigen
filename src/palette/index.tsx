export * from "./atoms"
export * from "./organisms"
export * from "./elements"
export * from "./Theme"
