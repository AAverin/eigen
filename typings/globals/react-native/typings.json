{
  "resolution": "main",
  "tree": {
    "src": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts",
    "raw": "registry:dt/react-native#0.29.0+20160719232534",
    "typings": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts"
  }
}
