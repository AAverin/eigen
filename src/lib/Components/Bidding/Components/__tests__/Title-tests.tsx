import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"

import { Title } from "../Title"

import { Theme } from "@artsy/palette"

it("renders properly", () => {
  const bg = renderer
    .create(
      <Theme>
        <Title>Confirm your bid</Title>
      </Theme>
    )
    .toJSON()
  expect(bg).toMatchSnapshot()
})
