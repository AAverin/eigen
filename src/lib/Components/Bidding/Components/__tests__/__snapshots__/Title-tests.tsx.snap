// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<Text
  className="Typography-sc-1pie2ao-0 ehDCiu"
  fontFamily="AGaramondPro-Semibold"
  fontSize="22px"
  lineHeight="28px"
  m={4}
  style={
    Array [
      Object {
        "fontFamily": "AGaramondPro-Semibold",
        "fontSize": 22,
        "lineHeight": 28,
        "marginBottom": 40,
        "marginLeft": 40,
        "marginRight": 40,
        "marginTop": 40,
        "textAlign": "center",
      },
    ]
  }
  textAlign="center"
>
  Confirm your bid
</Text>
`;
